src/components/SecondaryPanes/Frames/Group.css

```diff
@@ -14,24 +14,33 @@
   direction: ltr;
 }
 
-.frames [role="list"] .frames-group.expanded .group,
-.frames [role="list"] .frames-group.expanded .group .location {
+.frames-group.expanded .group,
+.frames-group.expanded .group .location {
   color: var(--theme-highlight-blue);
 }
 
-.frames [role="list"] .frames-group .frames-list [role="listitem"] {
-  padding-left: 30px;
+.frames-group .frames-list [role="listitem"] {
+  padding-inline-start: 32px;
```

Comment:
is the size change for better alignment?

Reply:
Yep, it works a bit better in Debugger's call stack.
Nothing important so I can revert it if it's a concern.

Reply:
Reverted it for now, we have some UI work in progress for Debugger panes so we can always revisit later.